charts/csi-baremetal-scheduler-extender/values.yaml

```diff
@@ -29,18 +29,26 @@ image:
 
 patcher:
   enable: true
-  manifest: /etc/kubernetes/manifests/kube-scheduler.yaml
+  platform: vanilla
```

Comment:
Please add comments what are the optional possible for `platform` parameter

Reply:
done